mysite/javascript/googleanalytics.js

```diff
@@ -0,0 +1,17 @@
+(function(i,s,o,g,r,a,m){ i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+ga('create', 'UA-84547-17', 'auto', { 'allowLinker': true });
```

Comment:
Not really ideal to hard-code this, can it not be added in the CMS? Or is this code actually used to host the demo site (so users would be able to change the GA code if it was in the CMS?)

If so, then it's better to leave it here I guess.